

<html>
  <body>
    <!-- ... -->

    <%= form_tag url_for(action: 'create'), method: "post" do %>

      <%= label_tag 'prénom' %>
      <%= text_field_tag 'prénom', @user.first_name %>
      <br /> <br />

      <%= label_tag 'nom (optionnel)' %>
      <%= text_area_tag 'nom', @user.last_name %>
      <br /> <br />

      <%= label_tag 'description (optionnel)' %>
      <%= text_field_tag 'description', @user.description %>
      <br /> <br />

       <%= label_tag 'city.id' %>
      <%= text_field_tag 'city.id', @user.city_id.to_i %>
      <br /> <br />

         <%= label_tag 'email' %>
      <%= text_field_tag 'email', @user.email %>
      <br /> <br />

         <%= label_tag 'age' %>
      <%= text_field_tag 'age', @user.age %>
      <br /> <br />

       <%= label_tag 'password' %>
      <%= password_field_tag 'password', @user.password %>
      <br /> <br />

      <%= submit_tag "Create user" %>

    <% end %>

    <!-- ... -->
  </body>
</html>


<% if @user.errors.any? %>
  <p class="alert alert-danger" role="alert"> Nous n'avons pas réussi à créer l'utilisateur pour la (ou les) raison(s) suivante(s) :</p>
  <ul>
    <% @user.errors.full_messages.each do |message| %>
      <li><%= message %></li>
      <%= puts @user.city_id %>
    <% end %>
  </ul>
<% end %>
<%= link_to 'Back', url_for(:back) %>
